### DISCUSSION TITLE
Update FleetDM from their Github

### DISCUSSION THREAD

**Discussion by chadstout1939** [Security-Onion-Solutions/securityonion#10264] [Unsupported Versions]
Good evening,

I was testing one of the newer Security Onion distributions (2.230) in a Standalone environment, and found the FleetDM/Osquery module wouldn't allow customization to its config to enable Users and Software collection, as found here:

https://github.com/Security-Onion-Solutions/securityonion/discussions/7844#discussioncomment-2647835

In my currently operating 2.200 deployment, these features work. It seems that they had disabled these features for Free users sometime between 2.200's and 2.230's release, and then re-enabled the features:

https://github.com/fleetdm/fleet/issues/9304 

Unfortunately, they fixed it just a few days after the latest update for FleetDM in late February. 

Would it be possible to get an update for FleetDM using their fix for this in one of the coming updates for the 2.3.X versions?

Thank you for all you guys do!

**Comment by dougburks** [comment]
Last week, we released Security Onion 2.3.250 which includes FleetDM 4.31.1:
https://blog.securityonion.net/2023/05/security-onion-23250-now-available.html
